.github/workflows/windows.yml

```diff
@@ -11,10 +11,7 @@ jobs:
   buildInContainer:
     runs-on: ubuntu-latest
     container:
-      # FIXME: Use templating once we can mangle org/repo name into something that docker accepts.
-      # and anonymus / actions access to github packages works.
-      #image: docker://docker.pkg.github.com/${{ github.repository }}/mxe-build-container:0.9
-      image: docker://dirkhh/mxe-build-container:0.9
+      image: docker://subsurface/mxe-build-container:1.0
```

Comment:
dockerhub shows only the `1.0.stage1` tag:
https://hub.docker.com/r/subsurface/mxe-build-container/tags

xref
https://github.com/Subsurface-divelog/subsurface/pull/2411/checks?check_run_id=325214658

Reply:
https://github.com/Subsurface-divelog/subsurface/pull/2411/checks?check_run_id=325214530

```
invalid argument "***/mxe-build-container-useNewContainers:1.0.stage1" for "-t, --tag" flag: invalid reference format: repository name must be lowercase
```

not sure where the `useNewContainers` is coming from.

Reply:
(a) only shows stage1 tag because the stage 2 did run until the previous PR was merged
(b) that's the branch name which @glance- suggested we include in the name... but it turns out the name needs to be all lower case. See commit I just pushed.